packages/hoppscotch-common/src/services/banner.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Service } from "dioc"
+import { ref } from "vue"
+
+export type BannerType = "info" | "warning" | "error"
+
+export type BannerContent = {
+  type: BannerType
+  text: string
+  // Can be used to display an alternate text when display size is small
+  alternateText?: string
+}
+
+export class BannerService extends Service {
```

Comment:
Provide a small sentence that defines what this Service does as a JSDoc comment.
See: https://github.com/hoppscotch/hoppscotch/blob/73b3ff8e411705dd842abb3088b1a20bc9aa7df5/packages/hoppscotch-common/src/services/interceptor.service.ts#L123-L127